src/utils/fileUtils.js

```
@@ -79,16 +79,27 @@ class FileUtils {
             const jsonFilePath = path.join(outputDir, `${baseFilename}.json`);
             fs.writeFileSync(jsonFilePath, JSON.stringify(data, null, 2));
             console.log(ColorUtils.green(`JSON report saved to: ${ColorUtils.yellow(jsonFilePath)}`));
+            this.checkFileSize(jsonFilePath);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)